perses/rjmc/geometry.py

```diff
@@ -536,9 +525,9 @@ def _propose_angle(self, angle, beta):
         Bond angle proposal
         """
         theta0 = angle.type.theteq*units.radians
-        k_eq = angle.type.k*units.kilocalories_per_mole/(units.radians**2)
-        sigma = beta*2.0/np.sqrt(2.0*k_eq/k_eq.unit)
-        theta = sigma/sigma.unit*np.random.random()*theta0.unit + theta0
+        k = angle.type.k*units.kilocalorie_per_mole/units.radian**2
```

Comment:
The AMBER force constants may also be off by a factor of two.  Trace the call @swails uses here carefully to double-check:
https://github.com/ParmEd/ParmEd/blob/master/parmed/structure.py#L2053-L2054